eng/Versions.props

```
@@ -12,6 +12,7 @@
     <!-- Packages from dotnet/core-setup -->
     <MicrosoftNETCoreAppPackageVersion>3.0.0-preview-27404-9</MicrosoftNETCoreAppPackageVersion>
     <!-- Packages from dotnet/corefx -->
+    <MicrosoftBclJsonSourcesPacakgeVersion>4.6.0-preview.19105.1</MicrosoftBclJsonSourcesPacakgeVersion>
```

```suggestion
    <MicrosoftBclJsonSourcesPackageVersion>4.6.0-preview.19105.1</MicrosoftBclJsonSourcesPackageVersion>
```

This variable has a typo everywhere it's used. The bot won't be able to update automatically with this typo.

FYI - this seems to be a common problem, so I'm adding a test for this in https://github.com/aspnet/Extensions/pull/1089

---

Yeah I copied this from the SignalR pr that had a typo I think, so that's why its been common :)